pkg/feature/types.go

```diff
@@ -27,6 +28,12 @@ type OAuth struct {
 	Hmac string
 }
 
+type AuthorinoConfigs struct {
+	CrName,
+	ContainerImage,
+	LabelSelectors string
```

Comment:
If it's used only in one place - lets hardcode it in the template instead? I am not sure we need to make it configurable at this point. That was too much of a speculative development on my end.

Reply:
OK, I'll hardcode it in the template.

Reply:
Done